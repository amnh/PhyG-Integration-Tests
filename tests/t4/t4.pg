--build(distance, rdwag), inputting "Exclude.txt" file
set(seed:1634561640)
read(fasta:"chel-16.fas")
read(exclude:"Exclude.txt")
build(distance, rdWag, best:10)
report("t4.csv", data, overwrite)
report("t4_cr.csv", crossrefs, overwrite)
report("t4_collapse.tre", newick, graphs, collapse, overwrite)
report("t4_nocollapse.tre", newick, graphs, nocollapse, overwrite)
report("t4.ss", tnt, overwrite)
report("t4_ia.txt", ia, overwrite)
report("t4_ascii.txt", ascii, overwrite)
report("t4_pd.csv", pairdist, overwrite)