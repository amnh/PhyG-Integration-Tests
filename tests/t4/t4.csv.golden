"Input data contained:"
,"12 terminal taxa"
,"1 input data files"
,"1 character blocks"
,"1 total characters"

"Terminal taxa:"
" ","Alentus","Amblypygid","Aportus","Centruoides","Hadrurus","Limulus","Mastigoproctus","Pauroctonus","Rhiphicephalus","Thelichoris"
" ","Thermobius","Vonones"

"Excluded taxa:"
" ","Americhernus","Gea","Hypochilus","Artemia"

"Renamed taxa:","New Name","Original Name"
" ","None","None"

"Index","Block","Name","Type","Activity","Weight","Prealigned","Alphabet","TCM"
"0","../data/chel-16.fas#0","../data/chel-16.fas#0","NucSeq","active","1.0","unaligned","-,A,C,G,T","[[0],[1,0],[1,1,0],[1,1,1,0],[1,1,1,1,0]]"
